docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -136,6 +136,40 @@ Using a custom namespace solves problems later on because you do not have to ove
 
 1. Wait until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
 
+## Generate some test metrics
+
+{{< docs/shared source="alloy" lookup="agent-deprecation.md" version="next" >}}
+
+The Grafana Mimir Helm chart can collect metrics, logs, or both, about Grafana Mimir itself. This is called _metamonitoring_.
+In the example that follows, metamonitoring scrapes metrics about Grafana Mimir itself, and then writes those metrics to the same Grafana Mimir instance.
+
+1. Deploy the Grafana Agent Operator Custom Resource Definitions (CRDs). For more information, refer to [Deploy the Agent Operator Custom Resource Definitions (CRDs)](https://grafana.com/docs/agent/<AGENT_VERSION>/operator/getting-started/#deploy-the-agent-operator-custom-resource-definitions-crds) in the Grafana Agent documentation.
```

Comment:
```suggestion
1. Deploy the Grafana Agent Operator Custom Resource Definitions (CRDs). For more information, refer to [Deploy the Agent Operator Custom Resource Definitions (CRDs)](https://grafana.com/docs/agent/latest/operator/getting-started/#deploy-the-agent-operator-custom-resource-definitions-crds) in the Grafana Agent documentation.
```

Reply:
We use version substitution syntax rather than just linking to "latest" per https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution.